codebuild/spec/buildspec_ubuntu_integ_awslc.yml

```diff
@@ -56,7 +56,6 @@ phases:
       - printenv
       - GCC_VERSION=4.8 $CB_BIN_DIR/s2n_codebuild.sh
       - GCC_VERSION=6 $CB_BIN_DIR/s2n_codebuild.sh
-      - GCC_VERSION=9 $CB_BIN_DIR/s2n_codebuild.sh
   post_build:
```

Comment:
Why not build with GCC9? Did this change somehow break it?

Reply:
I forgot to remove this build after I adding [the memcmp compiler test](https://github.com/awslabs/aws-lc/blob/736fce4b11417ea07ae0df06913d7e87a3cd58ca/crypto/compiler_test.cc#L178-L187).

No, all good before I removing the build. I tested `codebuild/spec/buildspec_ubuntu_integ_awslc.yml` in my personal account 4 times. `all good` is because `buildspec_ubuntu_integ_awslc.yml` builds awslc with Gcc6 only once where there are three s2n_codebuild.sh with diff compilers.

Reply:
I can revert this deletion because s2n uses gcc6 compiled awslc when testing with gcc9.

What do you think?

Reply:
I'd remove it. The buildspec change is causing the CI not to run, so removing it will unblock the rest of your changes.

You can make the buildspec change separately.

Reply:
Removed. Thanks for explanation.